### PR TITLE
Styling quiz results/ml

### PR DESCRIPTION
**Pull request by maddielaw**
## Purpose:

*What does this merge do? Check all that apply.*
- [ ] New feature (non-breaking change which adds functionality)
- [X] Improves an existing feature
- [ ] Fixes a bug
      - Bug fixed:
- [ ] Refactor
      - Code refactored:
- [ ] Documentation update

## Description: What does this PR do?
This PR adds doughnut charts and some additional styling

## Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [X] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/QuizResults/QuizResults.js [hunk 1]
```diff
@@ -26,16 +27,19 @@ const QuizResults = () => {
         </div>
         <section className="result-tile-container">
           <div className="result-tile">
-            <p className="percentage">{repData.currentQuizResult.planned_parenthood}%</p>
-            <p>in support of legalizing abortion federally</p>
+            <DoughnutChart percentage={repData.currentQuizResult.planned_parenthood} issue={"Abortion"}/>
+            {/* <p className="percentage">{repData.currentQuizResult.planned_parenthood}%</p>
```
Comment (zliibbe) [inline src/components/QuizResults/QuizResults.js:31]: These comments could be deleted

**Review by nvalentini21 — APPROVED** [review]
This looks great - thanks for adding the charts! Now just have to get the tests up-to-date with the new design.

**Review by zliibbe — APPROVED** [review]
Maddie, these changes look great!

**Comment by JessFatta** [comment]
Looks great Maddie! Love the addition of the charts!

**Pull request merged**